<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UrTUpdater</class>
 <widget class="QMainWindow" name="UrTUpdater">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>495</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>495</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>495</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Urban Terror Updater</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/urtupdater.png</normaloff>:/images/urtupdater.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="windowFilePath">
   <string/>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/background_urtupdater.png); background-color:black; background-repeat: no-repeat; background-position: 20px 20px;</string>
   </property>
   <widget class="QStackedWidget" name="layouts">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>650</width>
      <height>495</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background: transparent; </string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="l_information">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QTextEdit" name="info_textZone">
      <property name="geometry">
       <rect>
        <x>121</x>
        <y>62</y>
        <width>508</width>
        <height>370</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>508</width>
        <height>370</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background: white;</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt;&quot;&gt;Loading, please wait...&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="l_download">
     <widget class="QProgressBar" name="dl_ProgressBar">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>235</y>
        <width>430</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;</string>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QLabel" name="dl_Text">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>205</y>
        <width>430</width>
        <height>13</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;</string>
      </property>
      <property name="text">
       <string>Downloading ...</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="l_move">
     <widget class="QLabel" name="mv_Text">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>205</y>
        <width>430</width>
        <height>13</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;</string>
      </property>
      <property name="text">
       <string>Moving ...</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="mv_ProgressBar">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>235</y>
        <width>430</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;</string>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="l_remove">
     <widget class="QLabel" name="rm_Text">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>205</y>
        <width>430</width>
        <height>13</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;</string>
      </property>
      <property name="text">
       <string>Deleting ...</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="rm_ProgressBar">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>235</y>
        <width>430</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;</string>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="l_audit">
     <widget class="QLabel" name="audit_Text">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>205</y>
        <width>430</width>
        <height>13</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;</string>
      </property>
      <property name="text">
       <string>Collecting information ...</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="audit_ProgressBar">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>235</y>
        <width>430</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;</string>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>true</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="l_waitAPI">
     <widget class="QLabel" name="wait_Text">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>205</y>
        <width>430</width>
        <height>13</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;</string>
      </property>
      <property name="text">
       <string>Contacting urbanterror.info for update data ...</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="Copyright">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>475</y>
      <width>545</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white; background: none;</string>
    </property>
    <property name="text">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Copyright Â© 2013 FrozenSand/0870760 B.C. Ltd&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="BT_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>437</x>
      <y>440</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: none; </string>
    </property>
    <property name="text">
     <string>BT2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BT_3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>440</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: none; </string>
    </property>
    <property name="text">
     <string>BT3</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="BT_1">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>334</x>
      <y>440</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: none; </string>
    </property>
    <property name="text">
     <string>BT1</string>
    </property>
   </widget>
   <zorder>Copyright</zorder>
   <zorder>layouts</zorder>
   <zorder>BT_2</zorder>
   <zorder>BT_1</zorder>
   <zorder>BT_3</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
